.github/workflows/pr-tests.yml

```
@@ -76,3 +76,33 @@ jobs:
           working-directory: ./packages/fuelet-wallet
           json-summary-path: ./coverage/coverage-summary.json
           json-final-path: ./coverage/coverage-final.json
+  tests-e2e-connectors:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Install dependencies for React app
+        run: pnpm install
+        working-directory: ./examples/react-app
+
+      - name: Install Playwright dependencies
+        run: pnpm install
+        working-directory: ./e2e-tests
+
+      - name: Install Playwright browsers
+        run: pnpm exec playwright install
+        working-directory: ./e2e-tests
+
+      - name: Run Playwright tests
+        run: pnpm exec playwright test
+        working-directory: ./e2e-tests
```

- no needs to run pnpm install
- pnpm can be installed in setup-node
- install playwright with chromiun
- execute tests from script on root
- execute using "xvfb-run --auto-servernum -- " due to linux issues

```suggestion
      - name: Set up Node.js
        uses: actions/setup-node@v3
        with:
          node-version: 20.11.0
          pnpm-version: 8.9.2

      - name: Install Playwright browsers
        run: pnpm exec playwright install  --with-deps chromium
        working-directory: ./e2e-tests

      - name: Run Playwright tests
        run: xvfb-run --auto-servernum -- pnpm test:e2e
        working-directory: ./e2e-tests
```